### DISCUSSION TITLE
Any example on How to setup an old router with openwrt that blocks all internet traffic but lets lan traffic through? Thx

### DISCUSSION THREAD

**Discussion by kocoman1** [GerryDazoo/Slinger#627] [General]
Any example on How to setup an old router with openwrt that blocks all internet traffic but lets lan traffic through? Thx

**Comment by flipper2222222** [comment]
you can block internet traffic by  MAC  address,  this might help
https://forum.openwrt.org/t/filter-internet-access-by-mac/104020
